NAS2D/Resource/Font.cpp

```diff
@@ -99,21 +99,21 @@ Vector<int> Font::glyphCellSize() const
 }
 
 
-Vector<int> Font::size(std::string_view string) const
+Vector<int> Font::size(std::string_view text) const
```

Comment:
Mildly curious why you find `text` is less ambiguous than `string`. I find them equivalent. I suppose this is perhaps due to assumptions about embedded newlines?

The closest I can think of is "Text Controls", for multiline text, though I'm also used to hearing "Text Box" for single line of text. There's not really a clear distinction in my mind here.

Reply:
It's a semantics thing: Fonts are applied to text (that doesn't change); strings can be manipulated. Additionally, yes, text implies multi-line more strongly than string.